### PR TITLE
Feature/api data cag

### LINKED ISSUE #1: Compile data from NYT books API
compiling seed data- data that we will populate our database with

### PR DESCRIPTION
**Pull request by cagallo**
## What Features are Being Added?

- Create bookData file with entire book objects from NYT API
- Create filteredBookData file with filtered book objects
- Create utils.js with generateAllBooks function to map all filtered books data 
- Install Express.js 

## What is Being Refactored?

- N/A

## Where should the reviewer start?

- All files listed above

## How should this be manually tested?

- npm install 
- node data/utils.js

## What are the relevant tickets?

- Create book data file from NYT books API

## Any Other Notes/Questions/Problems?

- None, thanks y'all!

### CODE REVIEW

File: data/filtered-books-data.js [hunk 1]
```diff
@@ -0,0 +1,724 @@
+const filteredBooks = [
```
Comment (krdurrett) [inline data/filtered-books-data.js:1]: These modified book objects look great!

File: data/utils.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const books = require('./books-data');
```
Comment (krdurrett) [inline data/utils.js:1]: This function to map through the larger data file works perfectly

**Comment by ectweitmann** [comment]
Nice work Chez everything looks great!

**Review by krdurrett — APPROVED** [review]
This is off to a great start

**Pull request merged**